.github/ISSUE_TEMPLATE/infrastructure-request-form-default.yml

```diff
@@ -0,0 +1,92 @@
+name: Infrastructure Request Form WIP
+description: Request new infrastructure or change an existing configuration.
+title: "[Platform Infrastructure]: Infrastructure setup and configuration for TEAM"
+labels: ["infrastructure-request"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Request new infrastructure or change an existing configuration. _All answers are optional._ The Platform Infrastructure team will work with you to gather all necessary requirements before processing your request. Providing information upfront can help to expedite your request. 
+  - type: input
+    id: team-name
+    attributes:
+      label: Team name
+      description: What is your team's name?
+      placeholder: e.g. Infrastructure Team
+    validations:
+      required: false
+  - type: input
+    id: application-name
+    attributes:
+      label: Application name
+      description: What is the name of your applicaiton or service?  
+      placeholder: e.g. Vets-API _or_ 1095-B
+    validations:
+      required: false
+  - type: input
+    id: application-repository
+    attributes:
+      label: Software repository
+      description: Please provide a link to your application's source-code repository  
+      placeholder: e.g. https://github.com/department-of-veterans-affairs/<my-repo>
+    validations:
+      required: false
+  - type: input
+    id: product-owner
+    attributes:
+      label: Product owner
+      description: Who is your team's product owner?
+      placeholder: e.g. Saima Simran
+    validations:
+      required: false
+  - type: input
+    id: product-manager
+    attributes:
+      label: Product manager
+      description: Who is your team's product manager?
+      placeholder: e.g. Hannah Ilja
+    validations:
+      required: false
+  - type: input
+    id: related-issues
+    attributes:
+      label: Link to initiative or epic 
+      description: Please share related issue(s) for additional context   
+      placeholder: e.g. https://github.com/department-of-veterans-affairs/va.gov-team/issues/<my-issue>
```

Comment:
This placeholder gets truncated. Is that okay or should we change it?

![image](https://user-images.githubusercontent.com/10993987/156580607-745772c5-46db-43f4-bb0e-4bbdf7bb0126.png)